core/block_validator_test.go

```diff
@@ -35,7 +35,7 @@ func testChainConfig() *ChainConfig {
 		Forks: []*Fork{
 			&Fork{
 				Name:  "Homestead",
-				Block: big.NewInt(1150000),
+				Block: big.NewInt(0),
```

Comment:
I was just fixing test for existing code. ecip-1010 in different branch, so we'll add test only there